### ISSUE TITLE
Add feature to only show unread messages 

### ISSUE THREAD

**Issue by Vollbrecht** [vector-im/element-meta#2193]
#### What would you like to do?
I would be able to only see unread messages, either in the maintab in a chatroom or if not possible at-least only unread messages in the `Thread` tab.
#### Why would you like to do it?
People in bigger chatrooms get a lot of potential unread messages, often it is really hard to find what part's are not read, people just get a notification that something is unread - but no way to see what is actually unread. That is really frustrating. I don't mind lots of notification.
#### How would you like to achieve it?
Have an extra option at top to only show unread messages. If thats not feasable for the main chat it would at least be nice to have the options in the `Thread` tab to filter by `unread messages`. Current options are only `show all threads` or `show my threads`

**Comment by natrius** [comment]
Agree, its kinda important to simply add an additional filter for "Unread Threads" like this:

![image](https://github.com/element-hq/element-meta/assets/7243447/91a44ee8-aa3e-422c-85c9-45ee9658cd34)
